test/test_diff_1.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env sh
```

Comment:
`test_diff_{1,2,3}.sh` are almost the same, therefore the code can be factored out.